### ISSUE TITLE
How to change the default options in Pop_OS-current.conf

### ISSUE THREAD

**Issue by wmutschl** [pop-os/pop#909]
Hi, I installed POP!_OS with BTRFS-inside-LUKS and everything works just fine. However, I created a subvolume for @ for / and need to pass some rootflags to the Pop_OS-current.conf such that everything boots up fine. That is, I need to always have the following Pop_OS-current.conf file:

```
title Pop!_OS
linux /EFI/Pop_OS-UUIDNUMBERCRYPT/vmlinux.efi
initrd /EFI/Pop_OS-UUIDNUMBERCRYPT/initrd.img
options root=UUID=UUIDNUMBERCRYPT ro rootflags=subvol=/@ quiet loglevel=0 systemd.show_status=false splash

```
But if I run `update-initramfs -u -k all`, the `options root` is changed back to its default
```
options root=UUID=UUIDNUMBERCRYPT ro quiet loglevel=0 systemd.show_status=false splash
```
 without my `rootflags=subvol=/@`. Is there a way to always include this flag to the conf file?

**Comment by wmutschl** [comment]
Okay, I think I found a way and would like to know whether this is a good one? I edited the "user" section of `/etc/kernelstub/configuration` to also include `rootflags=subvol=@` as "kernel_options". And now whenever I run e.g. `updateinitramfs -u -k all` the rootflags are also written in the conf file for the systemd boot entry `/boot/efi/loader/entries/Pop_OS-current.conf`.

Is this a good solution or will this be overwritten by updates?
